src/vsr.zig

```diff
@@ -1189,13 +1417,20 @@ test "quorums" {
     }
 }
 
+/// Set of replica_ids of cluster members, where order of ids determines replica indexes.
+///
+/// First replica_count elements are active replicas,
+/// then standby_count standbys, the rest are zeros.
+/// Order determines ring topology for replication.
+pub const Members = [constants.nodes_max]u128;
```

Comment:
I am wondering if I should wrap this in a struct just to have something to put methods to, instead of doing c-style `members_this`, `members_that`. Decided to stick to C-style, as that's simpler